projects/1.16-fabric/assets/advanced-xray-fabric-edition/advanced-xray-fabric/lang/zh_cn.json

```
@@ -0,0 +1,55 @@
+{
+	"category.xray": "Advanced XRay",
+	"keybinding.enable_xray": "启用XRay",
+	"keybinding.open_gui": "打开GUI",
+	"message.xray_active": "XRay已激活",
+	"message.xray_deactivate": "XRay已关闭",
+	"spacer": "-------------------------------------------------------",
+	"comment_forge": "Forge mod old lang stuff, please never translate",
+	"xray.single.cancel": "取消",
+	"xray.single.add": "添加",
+	"xray.single.delete": "删除",
+	"xray.single.save": "保存",
+	"xray.single.tools": "工具",
+	"xray.single.close": "关闭",
+	"xray.single.search": "搜索",
+	"xray.single.help": "帮助",
+	"xray.overlay": "XRay激活",
+	"xray.color.red": "红：",
+	"xray.color.green": "绿：",
+	"xray.color.blue": "蓝：",
+	"xray.input.gui": "GUI中的名字",
+	"xray.input.add": "添加方块",
+	"xray.input.add_hand": "添加手持的方块",
+	"xray.input.add_look": "添加目视的方块",
+	"xray.input.distance": "检测范围：%s",
+	"xray.input.show-lava": "显示熔岩：%s",
+	"xray.input.toggle_oredict": "匹配矿物词典",
+	"xray.title.config": "配置方块",
+	"xray.title.edit": "编辑方块",
+	"xray.title.edit_meta": "编辑元数据",
+	"xray.message.missing_input": "你必须填写所有输入项",
+	"xray.message.already_exists": "此方块已添加到方块列表",
+	"xray.message.added_block": "已成功添加%s。",
+	"xray.message.updated_block": "方块列表已更新",
+	"xray.message.remove_block": "已删除%s",
+	"xray.message.block_exists": "%s已存在，请输入其他名称。",
+	"xray.message.unknown": "你尝试编辑的方块似乎不存在。",
+	"xray.message.removed": "已成功删除%s。",
+	"xray.message.config_missing": "未找到配置文件，正在创建中。",
+	"xray.message.invalid_hand": "%s不是方块",
+	"xray.message.thats_odd": "出错了？很抱歉，如果这种情况持续发生，请反馈此问题",
+	"xray.message.nothing_infront": "你目视的地方没有方块，对吧？",
+	"xray.message.not_a_number": "｛%s｝不是数字，请使用数字来指定元数据",
```

```suggestion
	"xray.message.not_a_number": "{%s}不是数字，请使用数字来指定元数据",
```
这个应该是占位符吧，需游戏内测试，是占位符则全文件修改